src/BloomExe/Publish/PublishView.cs

```diff
@@ -544,8 +557,12 @@ private void OnPrint_Click(object sender, EventArgs e)
 
 		private void FormActivatedAfterPrintDialog(object sender, EventArgs eventArgs)
 		{
+#if __MonoCS__
+			_pdfViewer.PrintFinished -= FormActivatedAfterPrintDialog;
```

Comment:
FormActivatedAfterPrintDialog ought to be renamed to something like PrintDialogClosed or PrintSetupCompleted